src/main/java/com/bhoruka/bloodbank/model/request/SearchCampRequest.java

```diff
@@ -0,0 +1,25 @@
+package com.bhoruka.bloodbank.model.request;
+
+import java.util.Date;
+
+import lombok.Builder;
+import lombok.Value;
+
+@Value
+@Builder
+public class SearchCampRequest {
+
+    private String partnerId;
+
+    private Date dateOfCamp;
+
+    private String expectedNoOfDonors;
+
+    private String actualNoOfDonors;
+
+    Integer pageNo;
+
+    Integer pageSize;
+
+    String sortBy;
```

Comment:
Can we use an enum with all the fields of CampModel and accept that here instead of String?